hnn_core/gui/gui.py

```diff
@@ -1530,23 +1534,38 @@ def add_drive_tab(params, log_out, drives_out, drive_widgets, drive_boxes,
                   tstop, layout):
     net = dict_to_network(params)
     drive_specs = net.external_drives
+    tonic_specs = net.external_biases
 
     # clear before adding drives
     drives_out.clear_output()
     while len(drive_widgets) > 0:
         drive_widgets.pop()
         drive_boxes.pop()
 
-    drive_names = sorted(drive_specs.keys())
+    drive_names = list(drive_specs.keys())
```

Comment:
This might cause a merge conflict with #851

Reply:
I rebased and resolved conflicts.